cli/glnexus_cli.cc

```diff
@@ -487,6 +487,64 @@ int main_dump(int argc, char *argv[]) {
     return 0;
 }
 
+// hard-coded configuration presets for unifier & genotyper. TODO: these
+// should reside in some user-modifiable yml file
+const char* config_presets_yml = R"eof(
+genotyper_config:
+  liftover_fields:
+    - orig_names: [DP, MIN_DP]
+      name: DP
+      description: '##FORMAT=<ID=DP,Number=1,Type=Integer,Description=\"Approximate read depth (reads with MQ=255 or with bad mates are filtered)\">'
+      type: int
+      combi_method: min
+      number: basic
+      count: 1
+    - orig_names: [SB]
+      name: SB
+      description: '##FORMAT=<ID=SB,Number=4,Type=Integer,Description=\"Per-sample component statistics which comprise the Fishers Exact Test to detect strand bias.\">'
+      type: int
+      combi_method: max
+      number: basic
+      count: 4
```

Comment:
It is conceivable that we can also default SB to zero for missing cases, the combi_method of max is a placeholder, we probably need to think more about what's the best approach.... maybe "average"?